distbelief/optim/downpour_sgd.py

```diff
@@ -20,11 +17,10 @@ def receive(self, sender, message_code, parameter):
         if message_code == MessageCode.ParameterUpdate:
             unravel_model_params(self.model, parameter)
 
-
 class DownpourSGD(Optimizer):
     """DownpourSGD"""
 
-    def __init__(self, params, lr=required, freq=required, model=required, internal_optim=None):
+    def __init__(self, params, lr=required, freq=required, model=required):
```

Comment:
Why are we removing the internal optimizer?

Reply:
I just took it out because i don't use it later